server/src/test/resources/registry.conf

```diff
@@ -0,0 +1,7 @@
+config {
```

Comment:
Is it necessary?
It seems that it is not consistent with the configuration file of the main project.

Reply:
same as above